### PR TITLE
Added ThingsBoard - Postgres

### PR DESCRIPTION
**Pull request by Fallstop**
### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v4/apps/thingsboard-postgres.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    variables:
+        - id: '$$cap_version'
+          label: ThingsBoard Version
+          defaultValue: '3.2.2'
+          description: Check out their Docker page for the valid tags hhttps://hub.docker.com/r/thingsboard/tb-postgres/tags
+          validRegex: "/^([^\\s^\\/])+$/"
+        - id: '$$mqtt_port'
+          label: MQTT device API port
+          defaultValue: '1883'
+          description: Chose what port to expose the MQTT device API on.
```
Comment (githubsaturn) [inline public/v4/apps/thingsboard-postgres.yml:25]: Chose => Choose

File: public/v4/apps/thingsboard-postgres.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          validRegex: '/^([0-9]{1,4}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$/'
+        - id: '$$coap_port'
+          label: COAP device API port
+          defaultValue: '5683'
+          description: Chose what port to expose the MQTT device API on.
```
Comment (githubsaturn) [inline public/v4/apps/thingsboard-postgres.yml:30]: Chose => Choose
and the description needs to be modified for CoAP port

Comment (Fallstop) [reply public/v4/apps/thingsboard-postgres.yml]: Such a stupid mistake to make, fixed!

**Comment by githubsaturn** [comment]
Thanks!

**Pull request merged**